docs/usage/usage-guide.md

```diff
@@ -183,20 +183,21 @@ Notice that we specifically call `state.concat()` to return a copied array with
 With `createReducer`, we can shorten that example considerably:
 
 ```js
-const todosReducer = createReducer([], {
-  ADD_TODO: (state, action) => {
-    // "mutate" the array by calling push()
-    state.push(action.payload)
-  },
-  TOGGLE_TODO: (state, action) => {
-    const todo = state[action.payload.index]
-    // "mutate" the object by overwriting a field
-    todo.completed = !todo.completed
-  },
-  REMOVE_TODO: (state, action) => {
-    // Can still return an immutably-updated value if we want to
-    return state.filter((todo, i) => i !== action.payload.index)
-  }
+const todosReducer = createReducer([], builder => {
```

Comment:
As we've had reports of confusion over "what is the recommended approach" with the usage guide still showing the object notation, I've update this part to builder notation.